tests/python/relay/strategy/arm_cpu/test_depthwise_conv2d.py

```diff
@@ -165,9 +165,13 @@ class TestDepthwiseConv2d_NHWC_HWOI_DSP(BasicDepthwiseConv2dTests):
         ((1, 3, 3, 256), (3, 3), 256, (1, 1), 1, 1),
         # Asymmetric height and width
         ((1, 25, 5, 64), (3, 3), 64, (1, 1), 1, 1),
+        # Larger kernel
+        ((1, 24, 24, 8), (5, 5), 8, (1, 1), 1, 1),
+        # Asymmetric kernel
+        ((1, 24, 24, 8), (3, 5), 8, (1, 1), 1, 1),
     )
```

Comment:
If we are relaxing the requirement of channels to be divisible by 4 for some cases, don't we want to test it as well?